nym-gateway-probe/src/lib.rs

```diff
@@ -159,6 +186,10 @@ pub struct Probe {
     direct_gateway_node: Option<DirectoryNode>,
     /// Pre-queried exit gateway node (used when --exit-gateway-ip is specified for LP forwarding)
     exit_gateway_node: Option<DirectoryNode>,
+    /// Localnet entry gateway info (used when --entry-gateway-identity is specified)
+    localnet_entry: Option<TestedNodeDetails>,
```

Comment:
rather than calling it "localnet", how about something that implies you provide explicit details yourself, i.e. you could very well run it on mainnet but provide all the information manually to avoid the extra queries